tornado/ioloop.py

```diff
@@ -440,7 +440,10 @@ def _setup_logging(self) -> None:
                 logging.getLogger("tornado.application").handlers,
             ]
         ):
-            logging.basicConfig()
+            # same as logging.basicConfig() but doesn't affect user's logger. issue #2961
+            logger = logging.getLogger("tornado")
+            if not logger.handlers:
```

Comment:
this check appears to be redundant with the condition immediately above (`if not any([..., logging.getLogger("tornado").handlers, ...]):`)

Reply:
(but may be irrelevant if this whole method is no longer needed since python-3.2)

Reply:
> this check appears to be redundant

@ploxiln Sorry I forgot about that. Fixed.

> this whole method is no longer needed

Since Tornado still works with Python 2.x I don't want to introduce a breaking change in this tiny PR.

Reply:
tornado-6.0 dropped support for python versions less than 3.5.2: https://www.tornadoweb.org/en/stable/releases/v6.0.0.html

Reply:
I suppose this PR does not break the default behavior in py3. What else am I missing?